src/query/compiler.js

```diff
@@ -310,9 +310,12 @@ assign(QueryCompiler.prototype, {
   counter() {
     const { counter } = this.single;
     const toUpdate = {};
-    toUpdate[counter.column] = this.client.raw(this.formatter.wrap(counter.column) +
-      ' ' + (counter.symbol || '+') +
-      ' ' + counter.amount);
+    Object.keys(counter).forEach((column) => {
+      let columnCounter = counter[column];
+      toUpdate[column] = this.client.raw(this.formatter.wrap(column) +
+        ' ' + (columnCounter.symbol || '+') +
```

Comment:
Why do we have `|| '+'` here? Seems like the `symbol` value is always set to `+` or `-` in `builder.js`.